BigBaseV2/src/backend/looped/self/godmode.cpp

```diff
@@ -15,5 +15,10 @@ namespace big
 
 			bLastGodMode = g->self.godmode;
 		}
+
+		if (g_local_player != nullptr)
+		{
+			g_local_player->m_damage_bits = g->self.proof_mask;
+		}
```

Comment:
Applying the proofs after god mode causes god mode to be disabled, since god mode uses `1 << 8`, either add god mode to the proof mask or call the god mode code after applying the mask.

Reply:
Fixed. Please check the latest commit.